packages/message-core/src/AvMessage.js

```diff
@@ -0,0 +1,116 @@
+class AvMessage {
+  subscribers = {};
+
+  constructor() {
+    this.isEnabled = true;
+    this.DEFAULT_EVENT = 'avMessage';
+    this.DOMAIN = /https?:\/\/([\w\d-]+\.)?availity\.(com|net)/;
+    window.addEventListener('message', ::this.getEventData);
```

Comment:
nit: I think we are trying to get away stage-0 stuff

Reply:
ah I didn't see that, just an artifact from copying from old angular lib, will update